content/applications/hr/payroll/contracts.rst

```diff
@@ -2,224 +2,373 @@
 
-Every employee in Odoo is required to have a contract in order to be paid. A contract outlines the
-terms of an employee's position, their compensation, their working hours, and any other details
-about their position.
+Every employee in Odoo is required to have a contract to be paid. A contract outlines the terms of
+an employee's job position, compensation, working hours, and any other details specific to their
+role.
 
 .. important::
    Contract documents (PDFs) are uploaded and organized using the *Documents* application, and are
-   signed using the *Sign* application. Ensure these applications are installed in order to send and
-   sign contracts. Please refer to the :doc:`/applications/productivity/documents` and
+   signed using the *Sign* application. Ensure these applications are installed to send and sign
+   contracts. Please refer to the :doc:`/applications/productivity/documents` and
    :doc:`/applications/productivity/sign` documentation.
 
-To view the employee contracts, go to the :menuselection:`Payroll app --> Employees --> Contracts`
-from the top menu. All employee contracts, and their current contract status, are displayed in a
-default kanban view. The default view displays both running contracts and contracts that need
-action. Expired and canceled contracts are hidden in the default view.
+To view the employee contracts, go to the :menuselection:`Payroll app --> Contracts --> Contracts`
+from the top menu. All employee contracts are displayed in a default list view, grouped into four
+categories: :guilabel:`New`, :guilabel:`Running`, :guilabel:`Expired`, and :guilabel:`Cancelled`.
+
+The default view has all categories collapsed. The number of contracts in each category is displayed
+next to the category name (example: `Running (20)`). To view the list of contracts under any
+category, click on the category, and the list expands downward, showing all contracts beneath it.
 
 .. image:: contracts/contracts-overview.png
    :align: center
    :alt: Contracts dashboard view showing running contracts and contracts with issues.
 
 .. note::
    The list of contracts in the *Payroll* application matches the list of contracts in the
-   *Employees* application. The default contracts view in the *Payroll* application displays running
-   contracts and contracts needing attention, while the default contracts view in the *Employees*
-   application displays all contracts in a kanban view, organized by their stage, regardless of
-   status. All contracts can be viewed by changing the filters.
-
-Create a new contract
-=====================
+   *Employees* application.
+
+.. _payroll/new-contract:
+
+Create new contracts
+====================
+
+For an employee to be paid, an active contract is required. When a new contract is needed, click the
+:guilabel:`New` button on the :guilabel:`Contracts` dashboard. A contract form appears where the
+information can be entered.
+
+.. _payroll/gen-info:
+
+General Information section
```

Comment:
```suggestion
General information section
```
since this is just what you are choosing to call this section, and not how any "title" of it appears on the UI, this can follow the traditional capitalization rule